### DISCUSSION TITLE
Only index.html page is displayed. and other pages have 404 error

### DISCUSSION THREAD

**Discussion by nutan-mali** [community/community#61555] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

hello, please help me I have always got this type of error. I push my files on the new repository and then add an index.html file. It's fine all but when I add about, home, and contact.HTML files in the same way as I upload index.html still I got a 404 error. Only the index page displays.

**Comment by iSemary** [comment, marked as answer]
Make sure that the other pages not listed in the .gitignore file in the root directory 
And if you could share any screenshots of the error you get it will help a lot

**Reply by nutan-mali** [reply]
Ok, I will try. And thanks for your reply.

**Reply by iSemary** [reply]
Anytime!
